src/app/components/card/sass/_variables.scss

```diff
@@ -0,0 +1,6 @@
+// COLORS
+$default-black :  rgba(0, 0, 0, 0.678);
+$default-white : #ffff;
+$color-orange-light: #ffb900;
+$color-orange-dark: #ff7730;
+$color-grey-dark : #3B3838;
```

Comment:
Looks like we removed the empty line here

Reply:
Thanks @CryceTruly , I worked on that